<launch>
  <!--    Launch coverage sequence generator node-->
  <rosparam command="load"
    file="$(find auto_mapping_ros)/config/auto_mapper_real.yaml" />
  <node pkg="auto_mapping_ros"
    name="coverage_sequence_creator_revit"
    type="coverage_sequence_creator_map"
    output="screen"/>
  <node pkg="map_service"
    name="map_server"
    type="revit_map_test"
    output="screen"/>
</launch>